File: backend/src/main/java/com/bluebee/smartsupply/model/Order.java
package com.bluebee.smartsupply.model;

import java.io.Serializable;
import java.sql.*;

public class Order implements Serializable {

    private Integer orderid;
    private Integer orderby;
    private Integer orderto;
    private Integer shopcd;
    private Timestamp orderdt;
    private Integer status;

    private String totOrderPrice;
    private Integer orderItmcnt;
    private String orderStatDesc;
    private String customerName;
    private String deliveryAddress;
    private String shopAddress;
    private String shopName;

    private Integer taskId;
    private Integer driverUserid;
    private Integer openTask;
    private Integer acceptstatcd;
    private Timestamp pickupts;
    private Integer deliverytype;
    private String driverDetails;

    public Timestamp getPickupts() {
        return pickupts;
    }

    public void setPickupts(Timestamp pickupts) {
        this.pickupts = pickupts;
    }

    public Integer getDeliverytype() {
        return deliverytype;
    }

    public void setDeliverytype(Integer deliverytype) {
        this.deliverytype = deliverytype;
    }

    public String getDriverDetails() {
        return driverDetails;
    }

    public void setDriverDetails(String driverDetails) {
        this.driverDetails = driverDetails;
    }

    public Integer getTaskId() {
        return taskId;
    }

    public void setTaskId(Integer taskId) {
        this.taskId = taskId;
    }

    public Integer getDriverUserid() {
        return driverUserid;
    }

    public void setDriverUserid(Integer driverUserid) {
        this.driverUserid = driverUserid;
    }

    public Integer getOpenTask() {
        return openTask;
    }

    public void setOpenTask(Integer openTask) {
        this.openTask = openTask;
    }

    public Integer getAcceptstatcd() {
        return acceptstatcd;
    }

    public void setAcceptstatcd(Integer acceptstatcd) {
        this.acceptstatcd = acceptstatcd;
    }

    public String getTotOrderPrice() {
        return totOrderPrice;
    }

    public void setTotOrderPrice(String totOrderPrice) {
        this.totOrderPrice = totOrderPrice;
    }

    public Integer getOrderItmcnt() {
        return orderItmcnt;
    }

    public void setOrderItmcnt(Integer orderItmcnt) {
        this.orderItmcnt = orderItmcnt;
    }

    public String getOrderStatDesc() {
        return orderStatDesc;
    }

    public void setOrderStatDesc(String orderStatDesc) {
        this.orderStatDesc = orderStatDesc;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public String getDeliveryAddress() {
        return deliveryAddress;
    }

    public void setDeliveryAddress(String deliveryAddress) {
        this.deliveryAddress = deliveryAddress;
    }

    public String getShopAddress() {
        return shopAddress;
    }

    public void setShopAddress(String shopAddress) {
        this.shopAddress = shopAddress;
    }

    public String getShopName() {
        return shopName;
    }

    public void setShopName(String shopName) {
        this.shopName = shopName;
    }

    public Integer getOrderid(){
    return this.orderid;
    }
    public void setOrderid(Integer orderid){ 
    this.orderid=orderid;
    }
    public Integer getOrderby(){ 
    return this.orderby;
    }
    public void setOrderby(Integer orderby){ 
    this.orderby=orderby;
    }
    public Integer getOrderto(){ 
    return this.orderto;
    }
    public void setOrderto(Integer orderto){ 
    this.orderto=orderto;
    }
    public Integer getShopcd(){ 
    return this.shopcd;
    }
    public void setShopcd(Integer shopcd){ 
    this.shopcd=shopcd;
    }
    public Timestamp getOrderdt(){ 
    return this.orderdt;
    }
    public void setOrderdt(Timestamp orderdt){ 
    this.orderdt=orderdt;
    }
    public Integer getStatus(){ 
    return this.status;
    }
    public void setStatus(Integer status){ 
    this.status=status;
    }


}

File: frontend/app/src/main/java/com/asuravan/smartsupply/pojo/Order.kt
package com.asuravan.smartsupply.pojo

import android.os.Parcel
import android.os.Parcelable
import java.sql.Timestamp

class Order():Parcelable {
    var orderid: Int? = null
    var orderby: Int? = null
    var orderto: Int? = null
    var shopcd: Int? = null
    var orderdt: Timestamp? = null
    var status: Int? = null

   var totOrderPrice: String? = null
   var orderItmcnt: Int? = null
   var orderStatDesc: String? = null
   var customerName: String? = null
   var deliveryAddress: String? = null
   var shopAddress: String? = null
   var shopName: String? = null

   var taskId: Int? = null
   var driverUserid: Int? = null
   var openTask: Int? = null
   var acceptstatcd: Int? = null

    var pickupts: Timestamp? = null
    var deliverytype: Int? = null
    var driverDetails: String? = null

    constructor(parcel: Parcel) : this() {
        orderid = parcel.readValue(Int::class.java.classLoader) as? Int
        orderby = parcel.readValue(Int::class.java.classLoader) as? Int
        orderto = parcel.readValue(Int::class.java.classLoader) as? Int
        shopcd = parcel.readValue(Int::class.java.classLoader) as? Int
        status = parcel.readValue(Int::class.java.classLoader) as? Int
        totOrderPrice = parcel.readString()
        orderItmcnt = parcel.readValue(Int::class.java.classLoader) as? Int
        orderStatDesc = parcel.readString()
        customerName = parcel.readString()
        deliveryAddress = parcel.readString()
        shopAddress = parcel.readString()
        shopName = parcel.readString()
        taskId = parcel.readValue(Int::class.java.classLoader) as? Int
        driverUserid = parcel.readValue(Int::class.java.classLoader) as? Int
        openTask = parcel.readValue(Int::class.java.classLoader) as? Int
        acceptstatcd = parcel.readValue(Int::class.java.classLoader) as? Int
        deliverytype = parcel.readValue(Int::class.java.classLoader) as? Int
        driverDetails = parcel.readString()
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeValue(orderid)
        parcel.writeValue(orderby)
        parcel.writeValue(orderto)
        parcel.writeValue(shopcd)
        parcel.writeValue(status)
        parcel.writeString(totOrderPrice)
        parcel.writeValue(orderItmcnt)
        parcel.writeString(orderStatDesc)
        parcel.writeString(customerName)
        parcel.writeString(deliveryAddress)
        parcel.writeString(shopAddress)
        parcel.writeString(shopName)
        parcel.writeValue(taskId)
        parcel.writeValue(driverUserid)
        parcel.writeValue(openTask)
        parcel.writeValue(acceptstatcd)
        parcel.writeValue(deliverytype)
        parcel.writeString(driverDetails)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Order> {
        override fun createFromParcel(parcel: Parcel): Order {
            return Order(parcel)
        }

        override fun newArray(size: Int): Array<Order?> {
            return arrayOfNulls(size)
        }
    }


}